docs/install/opensuse-installation-guide.md

```diff
@@ -3,19 +3,7 @@
 1. Install the Kata Containers components with the following commands:
 
    ```bash
-   $ source /etc/os-release
-   $ DISTRO_REPO=$(sed "s/ /_/g" <<< "$NAME")
-   $ [ -n "$VERSION" ] && DISTRO_REPO+="_${VERSION}"
-   $ DISTRO_REPO=$(echo $DISTRO_REPO | tr -d ' ')
-   $ ARCH=$(arch)
-   $ BRANCH="${BRANCH:-master}"
-   $ REPO_ALIAS="kata-${BRANCH}"
-   $ PUBKEY="/tmp/rpm-signkey.pub"
-   $ curl -SsL -o "$PUBKEY" "https://raw.githubusercontent.com/kata-containers/tests/master/data/rpm-signkey.pub"
-   $ sudo -E rpm --import "$PUBKEY"
-   $ zypper lr "$REPO_ALIAS" && sudo -E zypper -n removerepo "$REPO_ALIAS"
-   $ sudo -E zypper addrepo --refresh "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/${DISTRO_REPO}/" "$REPO_ALIAS"
-   $ sudo -E zypper -n install kata-runtime
+   $ sudo -E zypper -n install katacontainers
```

Comment:
Are we calling it `kata-runtime` on Centos/RHEL and `katacontainers` on openSUSE/SLES?

Reply:
@bergwolf Looks like it. I think its worthwile to have this consistent in the long run.

Reply:
On Fedora / CentOS / RHEL we made the decision of having one package for each component (kata-runtime, kata-shim, kata-osbuilder, kata-agent, ...). Looking at this decision now I clearly see how wrong we were. :-)

Hopefully we'll get things right for 2.0, and be consistent with the katacontainers name.
@etrunko, let's remember this for the future!